example/src/main/deploy/openshift/common.sh

```diff
@@ -11,6 +11,22 @@
 #    Red Hat - initial creation
 #    Bosch Software Innovations GmbH
 
+function prepare_openshift {
+    # creating the directory for Hono Server persistent volume
+    if [ ! -d /tmp/hono ]; then
+        mkdir /tmp/hono
+        chmod 777 /tmp/hono
+    else
+        echo /tmp/hono already exists!
+    fi
+
+    # creating Hono persistent volume (admin needed)
+    oc login $OPENSHIFT_MASTER -u system:admin $OPENSHIFT_MASTER
```

Comment:
are you sure you need to specify the URI twice?

Reply:
Thanks for spotting this. Fixed now